File: Components/eXpressApp Framework/XCRM/CS/XCRM.Module.Web/WebModule.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Updating;

namespace XCRM.Module.Web {
    [ToolboxItemFilter("Xaf.Platform.Web")]
    public sealed partial class XCRMAspNetModule : ModuleBase {
        public XCRMAspNetModule() {
            InitializeComponent();
            SiteMode = false;
        }
        public override IEnumerable<ModuleUpdater> GetModuleUpdaters(IObjectSpace objectSpace, Version versionFromDB) {
            return ModuleUpdater.EmptyModuleUpdaters;
        }
        public bool SiteMode { get; set; }
    }
}

File: Components/eXpressApp Framework/SimpleProjectManager/VB/SimpleProjectManager.Module.Web/WebModule.vb
Imports System.ComponentModel
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Updating

<ToolboxItemFilter("Xaf.Platform.Web")> _
Partial Public NotInheritable Class SimpleProjectManagerAspNetModule
    Inherits ModuleBase
    Public Sub New()
        InitializeComponent()
    End Sub
    Public Overrides Function GetModuleUpdaters(ByVal objectSpace As IObjectSpace, ByVal versionFromDB As Version) As IEnumerable(Of ModuleUpdater)
        Return ModuleUpdater.EmptyModuleUpdaters
    End Function
End Class
